qiskit/circuit/quantumcircuit.py

```diff
@@ -1110,6 +1151,65 @@ def ancillas(self) -> list[AncillaQubit]:
         """
         return self._ancillas
 
+    @property
+    def num_vars(self) -> int:
+        """The number of runtime classical variables in the circuit.
+
+        This is the length of the :meth:`iter_vars` iterable."""
```

Comment:
I guess not haha.  If we did, I suppose it'd have to have a special rule for one-line docstrings, since those would look really weird with a trailing newline.